helm-charts/falcon-sensor/values.yaml

```diff
@@ -188,7 +188,7 @@ container:
       # list. For example:
       #
       # namespaces: ns1,ns2,ns3
-      namespaces:
+      namespaces: ""
```

Comment:
This works. Could also set the following in `helm-charts/falcon-sensor/templates/container_secret.yaml` on line 15 that may or may not be better:

```
{{- $myns := split "," (.Values.container.image.pullSecrets.namespaces | default "") -}}
```

Thoughts?

Reply:
I would go on the values.yaml file as other tools (like argocd) are reading the values.yaml to build a UI didn't checked if it's decoding well or not on argocd but I'm fine with the default.

I think it's a helm issue so could impact other tools reading the values.yaml without looking at the schema.

Reply:
`{{- $myns := split "," (.Values.container.image.pullSecrets.namespaces | default "") -}}` should also solve your problem since the schema value and validation is `null` or `string` which means it can handle both and not error out. Schema validation will need to be fixed as well to be string only if pursuing updating the in-chart `values.yaml ` file.